dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/transform/strategy/CleanupStrategy.java

```diff
@@ -0,0 +1,36 @@
+package com.dotmarketing.portlets.contentlet.transform.strategy;
+
+import static com.dotmarketing.portlets.contentlet.transform.strategy.TransformToolbox.privateInternalProperties;
+
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.liferay.portal.model.User;
+import java.util.Map;
+import java.util.Set;
+
+public class CleanupStrategy extends AbstractTransformStrategy<Contentlet> {
+
+    CleanupStrategy(final TransformToolbox toolBox) {
+        super(toolBox);
+    }
+
+    @Override
+    public Contentlet fromContentlet(final Contentlet contentlet) {
+        return contentlet;
+    }
+
+    @Override
+    public Map<String, Object> transform(final Contentlet contentlet, final Map<String, Object> map,
+            final Set<TransformOptions> options, User user) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'user' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)